### DISCUSSION TITLE
Too slow

### DISCUSSION THREAD

**Discussion by lbpth** [QL-Win/QuickLook#1381] [General]
The app takes few seconds to open file. So it is not quick to be QuickLook. It should take only 0.5-1s to open file as a previewer. Please double check

**Comment by emako** [comment]
QL takes a few seconds to start up for the first time.

As for why, it is because it will prepare to decompress some resource files.

This is really not a good practice and needs to be optimized.

**Comment by emako** [comment]
Transfer to an issue and close this disscussion.
